app/api_calls/send_slack_message.liquid

```diff
@@ -0,0 +1,15 @@
+---
+to: "{{ context.constants.SLACK_WEBHOOK_URL }}"
+trigger_condition: "{% if context.constants.SLACK_WEBHOOK_URL %}true{% endif %}"
+format: http
+request_type: POST
+request_headers: '{
+  "Content-Type": "application/json"
+}'
+---
+{% capture message %}
+{{ form.actor.name }} posted a new question: <https://{{ context.location.host }}{{ form.object.url }}|{{ form.object.name }}>
```

Comment:
@damcik What is `form` in this context?

Reply:
https://documentation.platformos.com/developer-guide/notifications/creating-api-call-notification#step-1-create-api-call-notification